File: QRCode.FEW/ClientApp/src/app/models/qr_payment.ts
export interface qr_payment {
    qrpaymentid: number;
    packcode: string;
    packname: string;
    userid: number;
    qrenterpriseid: number;
    payment_date: Date;
    created_date: Date;
    created_by: number;
    lastcreated_date: Date;
    lastcreated_by: number;
}
export interface payment_view{
    qrpaymentid: number;
    packcode: string;
    packname: string;
    userid: number;
    email:string;
    phone:string;
    created_date: Date;
    payment_date: Date;
}
export interface package_objet {
    packcode: string;
    packname: string;
    price: number;
}

File: QRCode.Core/Domain/qr_payment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QRCode.Core.Domain
{
    public class qr_payment
    {
        public virtual long qrpaymentid { get; set; }
        public virtual string packcode { get; set; }
        public virtual string packname { get; set; }
        public virtual int userid { get; set; }
        public virtual int qrenterpriseid { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual int created_by { get; set; }
        public virtual DateTime? lastcreated_date { get; set; }
        public virtual int lastcreated_by { get; set; }
        public virtual DateTime? payment_date { get; set; }
    }
}
